### ISSUE TITLE
System color Palette in Configuration

### ISSUE THREAD

**Issue by miguelsotobaez** [cdgco/VestaWebInterface#74]
### OS / Environment

```
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.1 LTS
Release:        18.04
Codename:       bionic
Linux srv1.kumeweb.com 4.15.0-42-generic #45-Ubuntu SMP Thu Nov 15 19:32:57 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

### Summary of the problem
This is not a problem, just an enhancement, it would be a good idea to define the color template with a palette in system configuration. This would be good because we can change the color just like our branding company.

### Steps to reproduce the behavior
THere is no step right now to do this. Only in color selector like Blue, Purple, etc.

### Source code / logs / error code
Therer is no error code

**Comment by cdgco** [comment]
Good idea, I will try to add this to the next update. 

As of now, the best way to achieve this would be to edit the [/css/colors/blue.css](https://github.com/cdgco/VestaWebInterface/blob/master/css/colors/blue.css) file and change the variable values in the `/* Theme Colors */` section

This file is the easiest to change and has a preset variable for your primary color (`--main-theme-color: #2cabe3;`), secondary color (`--secondary-theme-color: #54667a;`), and so on.

You can also use any of the theme files as a template for your own theme and apply it by saving the file in the themes folder and changing the `THEME` value in your MySQL database to the custom file name in order to apply it.

**Comment by cdgco** [comment]
Feature Completed, to be published in v0.5.4.
